### ISSUE TITLE
Ballistic Shield moves us closer towards skill issue

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#50]
The new Ballistic Shield item is a net negative on the gameplay. It makes the game less about skill and more about having the right loadout (any grenade) or having more DPS (bulletspam guns). This is obviously an issue.

But, it could be fixed rather easily. Better yet, we could turn the shield into a perfect bulletspam counter.

The simple change is to change the way the shield blocks damage from
 - the shield blocks all damage that hits it

to
 - the shield blocks 7 damage from each shot that hits it.

This way, the shield would be very effective (although still not perfect) against assault rilfes (which tend to have damage in the 12-15 range, thus suffering a 50%-70% loss of damage), but would not be as useful against snipers (HS deals 30 damage per shot, which would drop to 23 when hitting a shield - which is *actually useful* as it means the user will survive 1 more shot than usually)

The actual flat damage decrease value is up to balancing, of course. It's also convenient as it makes balancing easier.

This is also arguably realistic as it reflects the fact that high caliber bullets, when they are going somewhere, **will go there** no matter how hard you try and stop them. A measly riot shield probably would not stop a bullet from an anti-materiel rifle; but would easily handle some low-caliber crap like pistol or SMG fire

**Comment by Gregory-Y** [comment]
good idea, but make it 8 so p50 can't deal any damage :))))

**Comment by deferred-impact** [comment]
Sure. Personally i would make it so that it decreases the damage by like 12 (making it fully counter most ARs), but i fear that certain bulletspam enthusiasts would get too upset about that

**Comment by Gregory-Y** [comment]
They don't seem to have any major complaints about shield countering automatic weapons so that's feasible.

**Comment by JacKDot2** [comment]
Crossbow deals no damage to shield (I mean wtf it shots arrows)

**Comment by JacKDot2** [comment]
Crossbow deals no damage to shield (I mean wtf it shots arrows)

**Comment by nlpha** [comment]
shotgun will be blocked hard, not just ar

**Comment by deferred-impact** [comment]
I don't see a problem with the shotgun being blocked. Every gun that fires tiny projectiles is either continuous spam (ARs, SMGs), or burst spam (shotguns). Don't you know how huge the spread of Pump shots is at the end of its range? Undeniably, at that range, the HS deserves to be rewarded more

**Comment by Gregory-Y** [comment]
@deferred-impact never said that the shields couldn't break, so I'd assume shotguns could still destroy the shield with concentrated fire. Therefore, shotguns getting completely blocked if the damage reduction is increased to like 10-12 is no problem.

What I do think should happen if we make the reduction 12 is an increase to the heavy sniper's damage to about 35 give or take so that it can consistently 3 shot a shield user.

**Comment by deferred-impact** [comment]
Of course, yes, the shield would still have limited HP (which is 120 right now, if you don't cycle), and bullets would still deal damage to the shield. It's just that a certain amount of that damage would also be applied to the person carrying the shield

**Comment by TheGoldenGemXD** [comment]
Good suggestion, just a little random idea I wanted to throw in, for 100% damage reduction, you would have to perfectly parry the bullet (similar to the idea some people had with blocking bullets with melee hits).

**Comment by Gregory-Y** [comment]
Sure but we can deal with that after shield cycling is nerfed. Anyway we can probably do the thing where we say that we reached a consensus on this thing
